### PR TITLE
Fix #17 + Possibly ESLint fix?

### LINKED ISSUE #17: Move dependencies to peer dependencies?
I was recently looking at the `package.json` and realized this package has 2 dependencies:
```
"@types/mssql": "^6.0.0",
"mssql": "^6.1.0"
```

I looked into this to make sure this is the right way to do things, and ran into [this](https://stackoverflow.com/questions/54926369/what-dependency-type-should-types-be-for-a-published-package) SO post where it seems like using `peerDependencies` would be better (at least for the type def).

Looking further I looked into one of the more popular express-session stores, connect-session-sequelize's [package.json](https://github.com/mweibel/connect-session-sequelize/blob/master/package.json) and found the "sequalize dependency" under peerDependencies.

As such it might make more sense to move these under peerDependencies instead of dependencies, as this package falls under the "plugins" type I keep hearing mentioned for use cases for peerDependencies.

Further reading:
- [differences between the different dependencies in package.json from a SO post](https://stackoverflow.com/questions/18875674/whats-the-difference-between-dependencies-devdependencies-and-peerdependencies)
- [nodejs docs on peer dependencies](https://nodejs.org/es/blog/npm/peer-dependencies/)

### PR DESCRIPTION
**Pull request by JLuboff**
I think we're okay to merge this unless you'd like to test a bit further. I've moved `@types/mssql` to `devDependencies`. 
Additionally, I have updated ESLint along with eslint config. If I'm in a file, and save it runs ESLint so I'm not positive if theres something else that needs to be changed for the linting in your hook?

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -759,10 +1227,23 @@
       "resolved": "https://registry.npmjs.org/@types/node/-/node-12.12.6.tgz",
       "integrity": "sha512-FjsYUPzEJdGXjwKqSpE0/9QEh6kzhTAeObA54rn6j3rR4C/mzpI9L0KNfoeASSPMMdxIsoJuCLDWcM/rVjIsSA=="
     },
+<<<<<<< HEAD
```
Comment (bradtaniguchi) [inline package-lock.json:1230]: Seems like there are merge conflicts in this, might be worth to delete the package-lock and re-install

Comment (JLuboff) [reply package-lock.json]: I marked this as resolved but not sure if that was the proper action to take. See the latest commit for the fix.

**Comment by bradtaniguchi** [comment]
The `npm run lint` command wasn't working for me, I think it was configured wrong initially. I was able to get it to work with changing it to:
```json
"lint": "npx eslint src --ext .ts"
```

Not sure why or how to make it work without specifying the `--ext` flag tho 🤔

**Comment by JLuboff** [comment]
Having the flag isn't too big a deal though. You need me to update `package.json` with that?

**Comment by bradtaniguchi** [comment]
Yea you can if you want, otherwise I can do it in a few minutes. Otherwise the PR LGTM :+1:

Just to check, you haven't "released" any of this to npm right?

**Comment by JLuboff** [comment]
I had it open so added it in the latest commit.

And correct, I have not published any updates to npm. Figured we get most of these knocked out than I can update semver and changelog

**Comment by bradtaniguchi** [comment]
Sounds good! You can merge whenever you want :)

**Pull request merged**